src/common/geometry.cc

```diff
@@ -0,0 +1,71 @@
+// BoB robotics includes
+#include "common/geometry.h"
+#include "common/macros.h"
+
+// Third-party includes
+#include "third_party/units.h"
+
+// Eigen
+#include <Eigen/Geometry>
+
+// Standard C includes
+#include <cmath>
+
+// Standard C++ includes
+#include <vector>
+
+namespace BoBRobotics {
+double
+distance2D(const Eigen::Vector2d &p1, const Eigen::Vector2d &p2)
+{
+    return hypot(p2.x() - p1.x(), p2.y() - p1.y());
+}
+
+
+double min(const Eigen::Matrix2d &line, int index)
```

Comment:
hah, even seeing the body of this, with my very limited grasp of eigen,  still don't know what it does :smile: